walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
I'm not seeing where this function is defined.

Reply:
This walkthrough example implemented these two functions in the same random-quotes directory. There weren't any changes to the file so it wasn't added/committed, but it was part of the walkthrough

Reply:
I see, my mistake.